spec/models/king_test.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe King, type: :model do
+  before :each do
+    @game = Game.new
+    @white_king = King.create(
+      x_coordinate: 2,
+      y_coordinate: 2,
+      game: @game,
+      color: "white"
+    )
+  end
+
+  describe "valid move for King piece" do
+    it "should return true if King moves one square horizontially" do
+      expect(@white_king.valid_move?(2, 1)).to eq true
+    end
+
+    it "should return true is King moves one square diagonally" do
+      expect(@white_king.valid_move?(3,3)).to eq true
+    end
+
+    it "should return false if King moves more than one square" do
+      expect(@white_king.valid_move?(6,7)).to eq false
+    end
+
```

Comment:
Ah, I can see. 2,7 passes, but 3,7 fails, because ONE of the values does equal 1 square move. I need to makes sure that BOTH of the moves cannot be more than one square. 
Thanks for catching that. I'll work on it today. :)